Lib/time.c

```diff
@@ -83,6 +83,9 @@ Time Time_now()
 		.ms    = systick_millis_count,
 		.ticks = ARM_DWT_CYCCNT,
 	};
+#elif defined(_sam_)
+	//SAM TODO
```

Comment:
systick_millis_count is available for SAM4S (it's a Cortex M4 feature)

ARM_DWT_CYCNT should be as well.